cli/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jboss.pnc.build.finder</groupId>
```

Comment:
The dot was to indicate that it is two words just like the dash in the project name. I know that that technically means there's a build package/group though.

Reply:
OK, as you wish :-)

Reply:
If you are OK with splitting it, I'd rather do that. To me, this more logically indicates that it's two words.

I know there's a side effect of making a build package, but then, maybe other build tools could live there at some point anyway.

Note that this will affect the Maven groupId and the Java package names.

Reply:
OK, Keep it split.